### DISCUSSION TITLE
Milvus collection indexes, loads, then seems to get corrupted and not loading again

### DISCUSSION THREAD

**Discussion by maustin10** [milvus-io/milvus#38364] [Q&A and General discussion]
I have a milvus collection that I can index, load, and then over time it seems to "de-load" making it unusable. Any suggestions of what to trouble shoot?

**Comment by yhmo** [comment]
Need the full log of the milvus.
If your milvus is a standalone deployed by docker, you can use "docker ps" to know the milvus container id, and "docker logs [container id]" to output the log.
If your milvus is a cluster in k8s, follow the doc to export the log: https://github.com/milvus-io/milvus/tree/master/deployments/export-log

**Reply by maustin10** [reply]
it is kubernetes milvus , here are the logs, thanks in advance! 
                                                                                                                                                                                    
[milvus-log.tar.gz](https://github.com/user-attachments/files/18120089/milvus-log.tar.gz)

**Comment by yhmo** [comment]
From the log, it seems there are three collections: hello_milvus, matter_291, test_collection
I only saw some errors that failed to search the 'matter_291'

`[2024/12/11 05:12:57.211 +00:00] [WARN] [proxy/task_search.go:740] ["failed to requery"] [traceID=2b9d6301ecf72bd137032d2ae397a365] [nq=1] [error="incomplete query result, missing id ec401bdb-dea3-4e19-9326-6aae93b081c7, len(searchIDs) = 10, len(queryIDs) = 5, collection=454528480040074106: inconsistent requery result"] [errorVerbose="incomplete query result, missing id ec401bdb-dea3-4e19-9326-6aae93b081c7, len(searchIDs) = 10, len(queryIDs) = 5, collection=454528480040074106: inconsistent requery result`

Usually, this error is caused by duplicate primary keys in the collection and delete actions to duplicated primary keys.
What do you mean "unusable"?
